backends-clickhouse/src/main/scala/org/apache/gluten/backendsapi/clickhouse/CHIteratorApi.scala

```diff
@@ -305,13 +311,24 @@ class CHIteratorApi extends IteratorApi with Logging with LogLevelUtil {
       }
     }
     var closed = false
+    val cancelled = false
```

Comment:
whta's usage of this var?

Reply:
Useless, just to keep the same style with previous codes...